src/Altinn.Platform/Altinn.Platform.Events/Tests/TestingControllers/EventsControllerTests.cs

```diff
@@ -84,6 +86,41 @@ public async void Post_GivenValidCloudEvent_ReturnsStatusCreatedAndCorrectData()
                 Assert.Contains(responseId, content);
             }
 
+            /// <summary>
+            /// Scenario:
+            ///   Post a valid CloudEvent instance.
+            /// Expected result:
+            ///   Returns HttpStatus Created and the Id for the instance.
+            /// Success criteria:
+            ///   The response has correct status and correct responseId.
+            /// </summary>
+            [Fact]
+            public async void Post_GivenValidCloudEvent_NotAuthorized()
+            {
+                // Arrange
+                string requestUri = $"{BasePath}/app";
+                string responseId = Guid.NewGuid().ToString();
+                CloudEvent cloudEvent = GetCloudEvent();
+
+                Mock<IEventsService> eventsService = new Mock<IEventsService>();
+                eventsService.Setup(s => s.StoreCloudEvent(It.IsAny<CloudEvent>())).ReturnsAsync(responseId);
```

Comment:
no need to set up the mock as it will never utilize it anyways. Let's save a few lines

Reply:
Need to rewrite mocking then. I keep this for a rainy day.